_submissions/round-12/0/2015-02-06-about-filipe_fernandes.md

```diff
@@ -7,6 +7,6 @@ permalink: /2015/02/filipe_fernandes/
 tags:
   - Biography
 ---
-[Filipe Fernandes](http://ocefpaf.github.io/homepage/) is a physical oceanographers interested in education and computer literacy.  During the day Filipe works for the Southeast Coastal Ocean Observing Regional Association ([SECOORA](http://ocefpaf.github.io/secoora/) and at night writes a [blog](http://ocefpaf.github.io/python4oceanographers/) about the use of python in oceanography.
+[Filipe Fernandes](http://ocefpaf.github.io/homepage/) is a physical oceanographers interested in education and computer literacy.  During the day Filipe works for the Southeast Coastal Ocean Observing Regional Association ([SECOORA](http://ocefpaf.github.io/secoora/)) and at night writes a [blog](http://ocefpaf.github.io/python4oceanographers/) about the use of python in oceanography.
```

Comment:
FYI, if you're using git from the shell (either linux or windows), you can set the line-wrapping option so that it's "correct" by doing the following command:
git config --global core.autcrlf=true